portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,107 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+class Place {
+  constructor(name, lat, lng, year, message = "") {
```

Comment:
[Nit] We should avoid shortening variable names like this.  Let's change 'lat' to 'latitude' and 'lng' to 'longitude'.